modules/document_repository/ajax/documentDelete.php

```diff
@@ -22,31 +32,44 @@
 
 $rid = $_POST['id'];
 
-$fileName = $DB->pselectOne("Select File_name from document_repository where record_id =:identifier",
-                            array(':identifier' => $rid));
-$userName = $DB->pselectOne("Select uploaded_by from document_repository where record_id =:identifier",
-                            array(':identifier'=> $rid));
-$dataDir  = $DB->pselectOne("Select Data_dir from document_repository where record_id =:identifier",
-                            array(':identifier'=> $rid));
+$fileName = $DB->pselectOne(
+    "Select File_name from document_repository where record_id =:identifier",
+    array(':identifier' => $rid)
+);
+$userName = $DB->pselectOne(
+    "Select uploaded_by from document_repository where record_id =:identifier",
+    array(':identifier' => $rid)
+);
+$dataDir  = $DB->pselectOne(
+    "Select Data_dir from document_repository where record_id =:identifier",
+    array(':identifier' => $rid)
+);
 
 $user =& User::singleton();
 
 //if user has document repository delete permission
 if ($user->hasPermission('document_repository_delete')) {
     $DB->delete("document_repository", array("record_id" => $rid));
     $msg_data['deleteDocument'] = $baseURL. "/document_repository/";
-    $msg_data['document'] = $fileName;
-    $msg_data['study'] = $config->getSetting('title');
-    $query_Doc_Repo_Notification_Emails = "SELECT Email from users where Active='Y' and Doc_Repo_Notifications='Y' and UserID<>:uid";
-    $Doc_Repo_Notification_Emails = $DB->pselect($query_Doc_Repo_Notification_Emails, array("uid"=>$user->getUsername()));
+    $msg_data['document']       = $fileName;
+    $msg_data['study']          = $config->getSetting('title');
+    $query_Doc_Repo_Notification_Emails = "SELECT Email from users".
```

Comment:
SQL keywords like 'from" or "SELECT" should be uppercase, table and column names should be the same case as the tablename in the schema